pyccel/codegen/printing/ccode.py

```diff
@@ -282,13 +282,88 @@ def _format_code(self, lines):
         return self.indent_code(lines)
 
     def _flatten_list(self, irregular_list):
-        if isinstance(irregular_list, (PythonList, PythonTuple)):
+        if isinstance(irregular_list, (PythonList, PythonTuple, list)):
```

Comment:
Is there any reason why `list` is added here?

Reply:
This function is no longer in use, I will remove it.